docs/section19/Rule19-18.md

```diff
@@ -0,0 +1,35 @@
+# Section 19 - Rule 19-18 
+**Schema Version:** 0.0.29
+**Mandatory Rule:** False   
+**Rule ID:** 19-18   
+**Rule Description:** For baseline systems 3 through 8, and 11, 12, and 13, the system fan electrical power for supply, return, exhaust, and relief shall be Pfan = bhp × 746/fan motor efficiency. Where, bhp = brake horsepower of baseline fan motor from  Table G3.1.2.9; fan motor efficiency = the efficiency from Table G3.9.1 for the next motor size  greater than the bhp using a totally enclosed fan cooled  motor at 1800 rpm
+**Rule Assertion:** Options are UNDETERMINED/NOT_APPLICABLE    
+**Appendix G Section:** G3.1.2.9        
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:** 
+1. Applies to baseline HVAC system types 3, 4, 5, 6, 7, 8, 11, 12, 13.
+
+**Function Calls:**  
+1. get_baseline_system_types()    
+2. baseline_system_type_compare()  
+
+## Rule Logic:   
+**Applicability Check 1**     
+Create a list of the target system types: `APPLICABLE_SYS_TYPES = [HVAC_SYS.SYS_3, HVAC_SYS.SYS_4, HVAC_SYS.SYS_5, HVAC_SYS.SYS_6, HVAC_SYS.SYS_7, HVAC_SYS.SYS_8, HVAC_SYS.SYS_11, HVAC_SYS.SYS_12, HVAC_SYS.SYS_13]`
+- Create dictionary of baseline system types: `baseline_system_types_dict_b = get_baseline_system_types(B_RMD)`  
+- For each hvac system in the B_RMD: `for hvac in B_RMD...HeatingVentilatingAirConditioningSystem:` 
+    - Get the baseline system type: `hvac_sys_type_b = list(baseline_system_types_dict_b.keys())[list(baseline_system_types_dict_b.values()).index(hvac.id)])`
+    - Check if system type is of an applicable system type:  `if any(baseline_system_type_compare(hvac_sys_type_b, target_sys_type, false) for target_system_type in APPLICABLE_SYS_TYPES):`
+    **Rule Assertion:**  
+    - Case 1: Baseline system is one of the applicable system types; outcome is UNDETERMINED: `outcome = UNDETERMINED and raise_message "For <insert hvac.id> perform manual check that the bhp used to model the baseline system fan power aligns with the calculation procedure in Table G3.1.2.9."`  
```

Comment:
Suggested message:
"For <insert hvac.id> perform manual check that fan motor efficiency established per Table G3.9.1 used for determining  baseline system fan power was based on the next motor size greater than the bhp using a totally enclosed fan cooled motor at 1800 rpm."`